URI/ByteString/Extension.hs

```diff
@@ -0,0 +1,53 @@
+{-# LANGUAGE FlexibleInstances #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+module URI.ByteString.Extension where
+
+import Data.ByteString (ByteString)
+import Data.String (IsString(..))
+import Data.Text (Text)
+import Data.Text.Encoding (decodeUtf8, encodeUtf8)
+import Lens.Micro
+
+import qualified Data.ByteString.Char8 as C8
+
+import URI.ByteString
+
+instance IsString Scheme where
```

Comment:
Are the `IsString` instances used outside of the tests?

Reply:
Yeah, for example: https://github.com/thoughtbot/yesod-auth-oauth2/pull/86/files#diff-c5023724ffce02c103ccf31e67ae3023R57